test/e2e-networking-library.sh

```diff
@@ -20,8 +20,14 @@ function install_istio() {
   fi
 
   if [[ -z "${NET_ISTIO_COMMIT}" ]]; then
-    # TODO: Figure out the commit of net-istio.yaml from net-istio.yaml
+    readonly NET_ISTIO_COMMIT=$(head -n 1 ${1} | grep "# Generated when HEAD was" | sed 's/^.* //')
```

Comment:
Test passed but it seems the script is not working as expected yet:

```
>> Installing Ingress
/home/prow/go/src/knative.dev/serving/test/e2e-networking-library.sh: line 23: NET_ISTIO_COMMIT: readonly variable
Got NET_ISTIO_COMMIT from ./third_party/istio-latest/net-istio.yaml:
/home/prow/go/src/knative.dev/serving/test/e2e-networking-library.sh: line 29: NET_ISTIO_COMMIT: readonly variable
Hard coded NET_ISTIO_COMMIT:
```

I am checking [upgrade log](https://prow.knative.dev/view/gs/knative-prow/pr-logs/pull/knative_serving/10073/pull-knative-serving-upgrade-tests/1326033967085785088).

Reply:
This should be fixed now.